### PR TITLE
csharp Add namespace

### PR DESCRIPTION
**Pull request by iesoftwaredeveloper**

### CODE REVIEW

**Comment by iesoftwaredeveloper** [comment]
I clearly am missing something.  Will cancel my updates and concede my failure for now.

**Comment by iHiD** [comment]
I'll fix it in a sec for you :)

**Comment by iHiD** [comment]
That should do it for you now!

**Comment by iHiD** [comment]
Thanks :)

File: test/languages/csharp_test.rb [hunk 1]
```diff
@@ -12,21 +12,25 @@ def test_full_example
...
               public static double Square(int i) => Math.Pow(2, i - 1);
 
               public static double Total() => Enumerable.Range(1, 64).Select(Square).Sum();
+            }
           }
         CODE
 
         expected = <<~CODE
-          public static class Grains
           {
```
Comment (ErikSchierboom) [inline test/languages/csharp_test.rb:27]: It would be nice if we could also remove these outer namespace curly braces

**Pull request merged**